### ISSUE TITLE
"Possibly confusing dependencies found" regression in 0.6.1

### ISSUE THREAD

**Issue by bitti** [clj-commons/aleph#676]
When running `lein deps :tree` in a project which includes aleph 4.6.1 as dependency, I get:

```
Possibly confusing dependencies found:
[aleph "0.6.1"] -> [org.clojure/tools.logging "1.1.0" :exclusions [org.clojure/clojure]]
 overrides
[aleph "0.6.1"] -> [manifold "0.3.0"] -> [org.clojure/tools.logging "1.2.4" :exclusions [org.clojure/clojure]]
```

So aleph includes a transitive dependency which depends on org.clojure/tools.logging 1.2.4 while having a direct dependency on org.clojure/tools.logging 1.1.0. I suggest updating the direct dependency to match the 1.2.4 version. This doesn't happen in aleph 4.6.0.

**Comment by KingMob** [comment]
Thanks for catching this

**Comment by bitti** [comment]
Thanks for fixing this. But do you know when you do a release which includes the fix?

**Comment by KingMob** [comment]
@bitti Sorry, been hard at work adding HTTP/2 support to Aleph. If it's critical, I can cut a quick patch release.

**Comment by bitti** [comment]
No problem, thanks for your hard work! Not it's not critical, I was just wondering if there is a timeline.

**Comment by KingMob** [comment]
@bitti A bunch of changes had accumulated, so I just released 0.6.2: https://clojars.org/aleph/versions/0.6.2
